src/pages/UitpasIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, BoxProps, getBoxProps } from '@/ui/Box';
+
+const UitpasIcon = (props: BoxProps) => {
```

Comment:
Would maybe put this in `ui` with the Icon components as it feels weird in `pages`